devices/tuya.js

```diff
@@ -3595,4 +3625,122 @@ module.exports = [
                 .withDescription('Turns the onboard LED on or off'),
         ],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_ztvwu4nk'}],
```

Comment:
Just to have a separate thread for discussion: this fingerprint is also used for at least another device with *different* data points (one of the Moes thermostats). The PR cannot be merged until this is sorted out, and I am afraid I just don't know where to go here.

At worst, I'll put this up somewhere as external converter and hope for a proper solution in the future.

Reply:
That would be strange (I've not seen this before), what datapoints are different and do we know the `_TZE200_ztvwu4nk` works correctly with the current converter?

Reply:
I did a quick check, among the overlapping data points, 20 has a different meaning (dead zone temp for BHT-002, fahrenheit min temp for HC-020), the others more or less agree except maybe `moesHold`. Quite a few data points just don't overlap (e.g. the schedule is different, plus the whole `thermostat_unit` thing does not exist in the BHT-002 converter). I will test the current converter again, but to my memory, HC-020 only half works, hence this PR.

I didn't keep notes, but my cursory search through old issues gave the impression that there are genuine BHT-002 devices with fingerprint `_TZE200_ztvwu4nk` which work with the current converter.

Reply:
PS: the X5H thermostat has more similar data points to HC-020, compared to the BHT-002.

Reply:
Could it be that this device was somehow mislabeled? (wrong manufacturer name flashed) Have you seen any others with the same thermostat as you have?

Reply:
> Could it be that this device was somehow mislabeled? (wrong manufacturer name flashed) Have you seen any others with the same thermostat as you have?

I only know of @KGS501 who seems to have the exact same device, down to the colour.

Somehow the Smart Life app can tell the difference. It is reported as "Zigbee Temperature Controller Public Version 2". MCU is V2.0.0. Maybe TuYa can recognise the MAC address on their end, which would not be that great for Z2M.

Reply:
@inext-gr  can you provide your data/database.db entry of your BHT-002-GCLZB ?